koans/1.intro.js

```diff
@@ -7,7 +7,7 @@ import {
 } from 'tape'
 
 import {
-  arrow,
+  arr,
```

Comment:
Agreed on 1,2, and 4.

About 3, I've been investigating a bit more and semantically there's no difference, but in practice in a language like Haskell when you write `Left 3` that is not a call to constructory thingy called Left that will return you a `Left 3`, but rather the actual value; `arr` however, is an actual constructor that returns an instance of an Arrow.

So yes, we shall name them both `Left/Right/Arrow`, since in this implementation, they are all functions that create instances of their types.

Idag vi lärt oss! 😄